src/pages/system-management/dict/list/Index.js

```diff
@@ -0,0 +1,389 @@
+        });
+    }
+
+    columns = [
+        {
+            title: '编码',
+            dataIndex: 'code',
+            editable: true,
+        },
+        {
+            title: '名称',
+            dataIndex: 'name',
+        },
+        {
+            title: '操作',
+            render: (text, record) => (
+                <Fragment>
+                    {record.children && (
+                        <span>
+                            <a onClick={() => this.handleItemModalVisible(true, record)}>新增明细</a>
+                            <Divider type="vertical" />
+                            <a onClick={() => this.handleUpdateModalVisible(true, record)}>编辑</a>
+                            <Divider type="vertical" />
+                            <Popconfirm
+                                title="你确定要删除此记录吗?"
+                                onConfirm={() => this.handleDelete(record)}
+                                okText="确定"
+                                cancelText="取消"
+                            >
+                                <a>删除</a>
+                            </Popconfirm>
+                        </span>
+                    )}
+                    {!record.children && (
+                        <span>
+                            <a onClick={() => this.handleUpdateItemModalVisible(true, record)}>编辑</a>
+                            <Divider type="vertical" />
+                            <Popconfirm
+                                title="你确定要删除此记录吗?"
+                                onConfirm={() => this.handleDeleteItem(record)}
+                                okText="确定"
+                                cancelText="取消"
+                            >
+                                <a>删除</a>
+                            </Popconfirm>
+                        </span>
+                    )}
+                </Fragment>
+            ),
+        },
+    ];
+
+    render() {
+        const {
+            dict: { data },
+            loading,
+        } = this.props;
+        const { selectedRows, modalVisible, modalItemVisible, updateModalVisible, updateItemModalVisible, dict, dictItem } = this.state;
+        const menu = (
+            <Menu onClick={this.handleMenuClick} selectedKeys={[]}>
+            </Menu>
+        );
+        const parentMethods = {
+            handleAdd: this.handleAdd,
+            handleModalVisible: this.handleModalVisible,
+        };
+        const parentItemMethods = {
+            handleAdd: this.handleAddItem,
+            handleModalVisible: this.handleItemModalVisible,
+        };
+        const parentUpdateMethods = {
+            handleAdd: this.handleUpdate,
+            handleModalVisible: this.handleUpdateModalVisible,
+        };
+        const parentUpdateItemMethods = {
+            handleAdd: this.handleUpdateItem,
+            handleModalVisible: this.handleUpdateItemModalVisible,
+        };
+        return (
+            <PageHeaderWrapper>
+                <Card bordered={false}>
```

Comment:
Codacy found an issue: [Expected indentation of 14 space characters but found 16.](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=6006324)